apache/superset
# fix(native-filter): Default value multi-select height in native filters

### SUMMARY
Fixes the height of the multi-select for the "Default value" in the native filters modal, as well as the multi-select in the native filters left sidebar.

### BEFORE
![119386138-f4dc2700-bc7b-11eb-834a-68c157121a35](https://user-images.githubusercontent.com/60598000/119506943-10782800-bd77-11eb-8952-4fc8d3be7f0d.png)


### AFTER
![Screen Shot 2021-05-25 at 16 29 06](https://user-images.githubusercontent.com/60598000/119506982-1837cc80-bd77-11eb-8e9d-c60e691c72d2.png)

![Screen Shot 2021-05-25 at 16 46 25](https://user-images.githubusercontent.com/60598000/119508886-d14ad680-bd78-11eb-8f13-7c0791f50cfe.png)


### TESTING INSTRUCTIONS
1. Open a dashboard
2. Select the filters
3. Add values in the "Default value" multi-select
4. Make sure the selected values are appearing correctly

### ADDITIONAL INFORMATION


- [x] Has associated issue: Fixes #14792
